### PR TITLE
Mainpagedesign

### PR DESCRIPTION
**Pull request by NolShep**
- Updated mainpage background and ui colors 
- Also tweaked some title names with mainpage and navbar

### CODE REVIEW

**Comment by NolShep** [comment]
Whoops might have broke something, says I have conflicts but I'm not understanding what the issue is???

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -69,3 +65,39 @@ input[type=submit]:hover{
 	display: inline-block; 
 	vertical-align: middle;  
 }
+
+
+/* Animation and Background colors */
+.background {
+  animation:slide 3s ease-in-out infinite alternate;
+  background-image: linear-gradient(-60deg, darkgrey 50%, black
```
Comment (zhaonian) [inline src/main/webapp/style.css]: can this fit in one line?

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -4,20 +4,13 @@
...
-}
-
 /* Style input fields */
 label {
 	display: block;
 	padding-top: 15px;
 	padding-left: 15px;
-	font-size: 30px;
+    font-size: 30px;
+
```
Comment (zhaonian) [inline src/main/webapp/style.css]: this and below indentations does not seem consistent

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -69,3 +65,39 @@ input[type=submit]:hover{
...
+  animation-duration:7s;
+}
+.background3 {
+  animation-duration:8s;
+}
+
+
+h1 {
+  font-family:monospace;
+}
```
Comment (zhaonian) [inline src/main/webapp/style.css]: nit: add one and only one new line in between the elements to be consistent?

**Review by zhaonian — CHANGES_REQUESTED** [review]
The issue is the changes on this branch is not based off the latest changes on main branch. Git could not figure out which version is the one we actually want to keep, so we need to resolve the conflicts and tell git this is the version we want.

You can either resolve the conflicts in command line, or do it here by tapping on the "Resolve conflicts" button. 

plz let me know if you need more help

File: src/main/webapp/style.css (hunk 2 above)
Comment (NolShep) [reply src/main/webapp/style.css]: weird... on my cloud shell file it shows them lined up

File: src/main/webapp/style.css (hunk 1 above)
Comment (NolShep) [reply src/main/webapp/style.css]: Yes, I'll fix it... prob was an accidental enter key haha

File: src/main/webapp/style.css (hunk 3 above)
Comment (NolShep) [reply src/main/webapp/style.css]: on it 👍

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -10,28 +10,34 @@
...
+        <li class="active"><a href="index.html">Home</a></li>
         <li class="active"><a href="#">Home</a></li>
         <!-- Add other html pages to href here to take you to new page-->
         <li><a href="#">Test Stations</a></li>
         <li><a href="#">Heat Map</a></li>
         <li><a href="#">COVID Data</a></li>
         <li><a href="#">Contacts</a></li>
-        </ul>
+        </ul>       
       </div>
```
Comment (zhaonian) [inline src/main/webapp/index.html:28]: indentation does not seem right.

**Review by zhaonian — APPROVED** [review]
Looks great! thanks 🙂

When you are ready to merge, could you do "Squash and merge" please so that we can keep the main branch's commit history brief?

**Pull request merged**